src/polymer/polymer-core-feature.ts

```diff
@@ -12,9 +12,21 @@
  * http://polymer.github.io/PATENTS.txt
  */
 
-import {ScannedPolymerElement} from '../polymer/polymer-element';
+import {Document, Feature, Resolvable, SourceRange, Warning} from '../model/model';
+import {ScannedPolymerProperty} from './polymer-element';
 
 /**
- * The metadata for a Polymer core feature.
+ * Scanned representation of a Polymer 1.x core "feature".
  */
-export interface ScannedPolymerCoreFeature extends ScannedPolymerElement {}
+export class ScannedPolymerCoreFeature extends Resolvable {
+  sourceRange: SourceRange|undefined;
+  warnings: Warning[] = [];
+
+  /** Properties this `_addFeature` call augments `Polymer.Base` with. */
+  properties: ScannedPolymerProperty[] = [];
+
+  resolve(_document: Document): Feature|undefined {
+    // TODO(aomarks) Implement.
```

Comment:
Yeah, I'll add the resolve logic to this PR and ping it again. Just wanted to get a review going on the first part.